### DISCUSSION TITLE
keycloak 23.0.5 dev mode | error when i save the configuration for a realm

### DISCUSSION THREAD

**Discussion by pereceptron** [keycloak/keycloak#26712] [Q&A]
Hello,
when I save the configuration for a realm changing the Access Token Lifespan the server crash.
First I get the next error: 
![image](https://github.com/keycloak/keycloak/assets/158502156/a26998dc-c327-4191-87be-0ae4e2fe3d97)
After (when i push "Press here to refresh and continue") I get the next error:
![image](https://github.com/keycloak/keycloak/assets/158502156/9be9a7ba-08de-449d-886c-c950752e8555)

I am getting this error in the commandline:
![image](https://github.com/keycloak/keycloak/assets/158502156/91e89fad-818c-404e-ae17-c5ca22693f99)

Any idea?
Thanks in advance

**Comment by bonnm** [comment]
You are not the first one:
https://github.com/keycloak/keycloak/issues/26665
https://github.com/keycloak/keycloak/issues/26597
There you will find advice how to repair the database.

**Comment by pereceptron** [comment]
Hello bonnm,
thanks for your answer. Ok, then seems a bug in last version. I will try with other previous versions.
Regards

**Comment by antikalk** [comment]
@pereceptron This should be fixed with 23.0.6. Give it a try :)
